### PR TITLE
Feature: show deprecated php warnings

### PR DESCRIPTION
**Pull request by mzk**
screen shot 2017-03-01 at 12 13 21

ref https://github.com/mzk/PHP-Parallel-Lint/pull/1

### CODE REVIEW

File: phpcs-ruleset.xml [hunk 1]
```diff
@@ -39,7 +39,7 @@
 
     <rule ref="Generic.Files.LineLength">
         <properties>
-            <property name="lineLimit" value="125"/>
+            <property name="lineLimit" value="135"/>
```
Comment (TomasVotruba) [inline phpcs-ruleset.xml]: I'd rather use multi-line constructor. This rule is quite important

Comment (mzk) [inline phpcs-ruleset.xml]:
```
public function __construct(
        PhpExecutable $phpExecutable,
        $fileToCheck,
        $aspTags = false,
        $shortTag = false,
        $deprecated = false)
    {
FILE: /private/var/www/PHP-Parallel-Lint/src/Process/LintProcess.php
----------------------------------------------------------------------
FOUND 2 ERRORS AFFECTING 2 LINES
----------------------------------------------------------------------
 29 | ERROR | [x] The closing parenthesis of a multi-line function
    |       |     declaration must be on a new line
    |       |     (PEAR.Functions.FunctionDeclaration.CloseBracketLine)
 30 | ERROR | [x] The closing parenthesis and the opening brace of a
    |       |     multi-line function declaration must be on the same
    |       |     line
    |       |     (PEAR.Functions.FunctionDeclaration.NewlineBeforeOpenBrace)
```
    public function __construct(
        PhpExecutable $phpExecutable,
        $fileToCheck,
        $aspTags = false,
        $shortTag = false,
        $deprecated = false
    )
    {
 31 | ERROR | [x] The closing parenthesis and the opening brace of a
    |       |     multi-line function declaration must be on the same
    |       |     line
    |       |     (PEAR.Functions.FunctionDeclaration.NewlineBeforeOpenBrace)
```
```
    public function __construct(
        PhpExecutable $phpExecutable,
        $fileToCheck,
        $aspTags = false,
        $shortTag = false,
        $deprecated = false
    ) {
Opening brace should be on a new line
    |       |     (Generic.Functions.OpeningFunctionBraceBsdAllman.BraceOnSameLine)
```
So, isn't possible with this ruleset

Comment (TomasVotruba) [inline phpcs-ruleset.xml]: I see. Crazy :D

**Comment by mzk** [comment]
ping @JakubOnderka

**Comment by JakubOnderka** [comment]
@mzk Sorry for late reply. Can you please resolve conflicts and rebase to current master? Then I will merge it and release new version

**Comment by mzk** [comment]
@JakubOnderka What took so long? Waiting for a conflict? Frequent release prevents conflicts.

**Comment by mzk** [comment]
Rebased, tests passed. @JakubOnderka

**Comment by mzk** [comment]
ping @JakubOnderka

**Comment by legoktm** [comment]
Thanks for adding this @mzk, I was going to file a bug for this feature request :-)

@JakubOnderka, would it be possible to have a new release/tag for this feature?

**Pull request merged**